### DISCUSSION TITLE
DataTables Warning Issue

### DISCUSSION THREAD

**Discussion by Ram-SN** [OHDSI/DataQualityDashboard#546] [Q&A] [answered]
Hi,

I am working on an effort in CU Anschutz to migrate our older VMs to new machine. In this effort, we ran into an issue when recreating the DQD web application. I was able to successfully trigger the dqd_script.r but when I publish it through the http-server as given in the documentation, an error about DataTables pops up. Quick context, we already a working instance of the dqdappweb
<img width="580" alt="dqd_error" src="https://github.com/OHDSI/DataQualityDashboard/assets/56275846/ab356349-d536-450c-9bd6-17165e9131d3">

**Comment by mgkahn** [comment]
Another potentially relevant bit of information to the above posting by Ram: Colorado recently started to include multiple rows in our CDM_SOURCE table because we wanted to capture ALL of the data sources that are going into our OMOP instance (EHR data, physician billing data, public health death records) since they have different update schedules. I do not know if this tidbit is relevant to the above error, but it is a just-implemented change to our CDM_SOURCE table. Thanks for any insights.

**Comment by katy-sadowski** [comment, marked as answer]
Hi there!  What version of DQD are you running?  This error looks like the the one reported [here](https://github.com/OHDSI/DataQualityDashboard/issues/439), but if you're on the latest version of DQD then this _should_ have been fixed.

The latest version of DQD also should not see any issues related to multiple rows in CDM_SOURCE; it chooses an arbitrary single row to grab the metadata.

**Reply by Ram-SN** [reply]
Hi Katy, I am upgrading from v2.1.2 to v2.6. Yes! it did turn out to be the same error, once we uniformly upgraded the application the dqd_script ran and displayed results on the UI successfully. We are currently updating our thresholds since the version upgrade looks to have brought changes to checks

**Reply by katy-sadowski** [reply]
Fantastic!  For a summary of check changes since 2.1.2, see here: https://ohdsi.github.io/DataQualityDashboard/news/index.html

And feel free to reach out with questions :)
